_IBC/crescent-axelar.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
We follow alphabetical order determining chain1 and chain2. Please change the order, axelar as chain 1 and crescent as chain 2 (and change file name).

Reply:
Done.